### PR TITLE
People-table-basics

### PR DESCRIPTION
**Pull request by DenisGurskiy**
https://DenisGurskiy.github.io/react_people-table-basics/

### CODE REVIEW

File: src/components/PeoplePage/PeoplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map((currentPeople: Person) => (
+                  <tr
```
Comment (volodymyr-soltys97) [inline src/components/PeoplePage/PeoplePage.tsx:82]: Use a destruction for `currentPeople`

File: src/components/PeoplePage/PeoplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    data-cy="person"
+                    className={classNames({
+                      'has-background-warning': slug === currentPeople.slug,
+                    })}
+                  >
+                    <td>
+                      <NavLink
+                        to={currentPeople.slug}
+                        className={classNames({
+                          'has-text-danger': currentPeople.sex === 'f',
```
Comment (volodymyr-soltys97) [inline src/components/PeoplePage/PeoplePage.tsx:93]: Consider will be better if you create a enum for all 'magic words' and use them everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Let's improve something

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**